src/test/java/task40/helpes/TableSortSearchPage.java

```diff
@@ -27,71 +26,67 @@ public WebElement getWebElementById(String id)
         return driver.findElement(By.id(id));
     }
 
-    public void collectTableDataList(String tableCssSelector, List<Employee> list)
+    public void collectTableDataList(String tableSelector, List<Employee> list)
     {
-        WebElement table = driver.findElement(By.cssSelector(tableCssSelector));
-        List<WebElement> rows =
-                table.findElements(By.tagName("tr"));
+        WebElement next = getWebElementById(nextButtonId);
+        boolean buttonEnabledState = true;
+        while (buttonEnabledState) {
+            WebElement table = driver.findElement(By.cssSelector(tableSelector));
+            List<WebElement> rows =
+                    table.findElements(By.tagName("tr"));
 
-        for (WebElement row : rows) {
-            List<WebElement> cols = row.findElements(By.tagName("td"));
-            String employeeName = "";     String employeePosition = "";   String employeeOffice = "";    int employeeAge = 0;     int employeeSalary = 0;
-            for (WebElement col : cols)
-            {
-                if (cols.indexOf(col)==0)
-                {
-                    employeeName = col.getText();
-                }
-                else  if (cols.indexOf(col)==1)
-                {employeePosition = col.getText();}
-                else  if (cols.indexOf(col)==2)
-                {employeeOffice = col.getText();}
-                else if (cols.indexOf(col)==3)
-                {
-                    employeeAge = Integer.valueOf(col.getText());
-                }
-                else if (cols.indexOf(col)==5){
-                    employeeSalary = Integer.valueOf(col.getText().replaceAll("[^0-9]",""));
-                }
+            for (WebElement row : rows) {
+                String employeeName = "";
+                String employeePosition = "";
+                String employeeOffice = "";
+                int employeeAge = 0;
+                int employeeSalary = 0;
```

Comment:
No need to initialize variable in such way. Such construction will also work:
 String employeeName = row.findElement(By.xpath("./td[1]")).getText();

Reply:
Done